src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -108,5 +108,50 @@ private static string LoadRules()
                     Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
                     "Rules/BuiltInRules.json"));
         }
+
+        /// <summary>
+        /// Loads a configurations file. If no file was passed, checks the default directory for this file.
+        /// </summary>
+        /// <param name="configurationsFilePath">The path to a configuration file to read.</param>
+        /// <returns>Configuration file path contents if a file exists.</returns>
+        private string GetConfigurationFileContents(FileInfo configurationsFilePath)
+        {
+            try
+            {
+                string configurationFileContents = configurationsFilePath == null ? null : File.ReadAllText(configurationsFilePath.FullName);
+                if (configurationFileContents == null)
+                {
+                    var defaultPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
+                        "/Configurations/Configuration.json");
```

Comment:
The usefulness of `Path.Combine()` here is so no path separation characters need to be included, and then it's safe across Linux and Windows.  Instead of writing out the path, just include each directory/file name as another argument to the function.
``` suggestion
                    var defaultPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
                        "Configurations", "Configuration.json");
```